src/components/H5PElement/H5PElement.jsx

```diff
@@ -41,16 +44,30 @@ class H5PElement extends Component {
     window.removeEventListener('message', this.handleH5PChange);
   }
 
-  handleH5PChange(event) {
+  async fetchH5PTitle(resourceId) {
+    const getEnv = getEnvironmentVariabel('NDLA_ENVIRONMENT', 'test');
+    const env = getEnv !== 'prod' ? '-' + getEnv : '';
+    const url = `https://h5p${env}.ndla.no/v1/resource/${resourceId}/copyright`;
```

Comment:
Det finnes en const h5pApiUrl i config. Det er bedre å bruke den direkte i stedet for å lage din egen.

Reply:
Og legg heller metoden inn i h5pApi.js.